docs/internal-documentation/maintenance/ReleaseProcess.md

```diff
@@ -146,8 +166,8 @@ git show-ref --tags
 
 -------
 
-Copyright (c) 2013, 2014, 2015, IIS (The Internet Infrastructure Foundation)  
-Copyright (c) 2013, 2014, 2015, AFNIC  
+Copyright (c) 2013, 2014, 2015, IIS (The Internet Infrastructure Foundation)
+Copyright (c) 2013, 2014, 2015, AFNIC
```

Comment:
Copyright (c) 2013-2017, IIS (The Internet Foundation in Sweden)
Copyright (c) 2013-2017, AFNIC